css/style.css

```diff
@@ -0,0 +1,524 @@
+.container {
+  background: url(../img/Background.png) no-repeat;
+  background-size: cover;
+  display: flex;
+  max-width: 1440px;
+  min-width: 320px;
+  min-height: 1062px;
+  margin: auto;
+}
+
+.menu { margin: 130px 100px 0; }
+
+.title {
+  color: #fff;
+  margin-bottom: 10px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 50px;
+  line-height: 59px;
+}
+
+.subtitle {
+  color: #fff;
+  margin: 0 0 60px 0;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 19px;
+  line-height: 22px;
+}
+
+.menu__select-level {
+  margin-bottom: 113px;
+  position: relative;
+}
+
+.level_btn {
+  display: flex;
+  margin-bottom: 23px;
+  position: relative;
+  background: none;
+  border: none;
+  color: #fff;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 50px;
+  line-height: 59px;
+  outline: none;
+  padding-left: 48px;
+}
+
+.level_btn::hover {
+  opacity: .8;
+  cursor: pointer;
+  position: relative;
+}
+
+.level_active::before {
+  content: '';
+  background-color: #d62b2e;
+  position: absolute;
+  top: 20px;
+  left: 0;
+  width: 20px;
+  height: 20px;
+  transform: rotate(45deg);
+}
+
+.start-button {
+  background: #d62b2e;
+  border: none;
+  border-radius: 7px;
+  color: #fff;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 37px;
+  margin: 0;
+  height: 95px;
+  width: 352px;
+  outline: none; 
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Olga_FinalGame_JS/pullRequest?prid=5836866)